destination/db/clickhouse.go

```diff
@@ -73,7 +77,20 @@ func GetClickHouseConnection(configuration map[string]string) (*ClickHouseConnec
 		log.Error(err)
 		return nil, err
 	}
-	return &ClickHouseConnection{conn}, nil
+	err = retry.OnNetError(func() error {
+		return conn.Ping(ctx)
```

Comment:
Why do you need this ping at all? For a centralized place to handle `this may indicate that incoming traffic is not allowed in the service networking settings` ?

Reply:
Yes. Cause otherwise, we will get this in the next select/insert/etc.